lms/djangoapps/grades/course_grade_factory.py

```diff
@@ -95,18 +94,9 @@ def iter(
             user=None, course=course, collected_block_structure=collected_block_structure, course_key=course_key,
         )
         stats_tags = [u'action:{}'.format(course_data.course_key)]
-        with self._course_transaction(course_data.course_key):
-            for user in users:
-                with dog_stats_api.timer('lms.grades.CourseGradeFactory.iter', tags=stats_tags):
-                    yield self._iter_grade_result(user, course_data, force_update)
-
-    @contextmanager
```

Comment:
**Note to reviewers:** Removed this `contextmanager` as it's not needed to clear the cache - since the cache is a per-request cache, which automatically gets cleared at the end of each request and each celery task.  I removed it since its presence indicated want for maintenance and called for unneeded clearing of other additional caches.